index.d.ts

```diff
@@ -0,0 +1,322 @@
+// Type definitions for sparqljs 3.1
```

Comment:
Just realized this file is added. Is it automatically generated? If so we probably shouldn't commit.

Reply:
It is not automatically generated, but without this in the package, the other repos are throwing errors. In sparqlalgebrajs and comunica they're asking for type definitions.
So is it ok if I commit it from the next iteration?

Reply:
How was it generated?